packages/material-ui/src/Box/styleFunction.js

```diff
@@ -72,6 +72,39 @@ const getThemeValue = (prop, value, theme) => {
   return { [prop]: value };
 };
 
+function createEmptyBreakpointObject(breakpoints) {
+  const breakpointsInOrder = breakpoints.keys.reduce((acc, key) => {
+    const breakpointStyleKey = breakpoints.up(key);
+    acc[breakpointStyleKey] = {};
+    return acc;
+  }, {});
+  return breakpointsInOrder;
+}
+
+function removeUnusedBreakpoints(breakpointKeys, output) {
```

Comment:
It's quite confusing to name a parameter `output` considering these are inputs. Especially when untyped we should strive for descriptive names.

Reply:
renamed to style